messages/rules-writer.md

```diff
@@ -1,3 +1,3 @@
 # error.unrecognized-file-format
 
-The output file %s has an unsupported extension. Valid extensions include: .json.
+The output file %s has an unsupported extension. Valid extension(s): .json.
```

Comment:
While I'm normally a fan of "word(s)", this is setting us up for when we have more valid extensions.